File: routeMap/app/src/main/java/ru/android/zheka/db/Trace.java
package ru.android.zheka.db;
import com.activeandroid.Model;
import com.activeandroid.annotation.Column;
import com.google.android.gms.maps.model.LatLng;
import com.google.android.gms.maps.model.PolylineOptions;

public class Trace extends Model{
	@Column(name = "name")
	public String name;
	@Column(name = "start")
	public LatLng start;
	@Column(name = "end")
	public LatLng end;
	//use , for lat-lng delimiter and ; for points delimiter
	@Column(name = "data")
	public DataTrace data;
	//public PolylineOptions data;
	public String toString(){
		UtilePointSerializer util = new UtilePointSerializer();
		UtileTracePointsSerializer util1 = new UtileTracePointsSerializer();
		return new StringBuilder().append("name:")
				.append(name)
				.append(" start:")
				.append((String)util.serialize(start))
				.append(" end:")
				.append((String)util.serialize(end))
				.append("data:")
				.append((String)util1.serialize(data)).toString();
	}
}

File: routeMap/app/src/main/java/ru/android/zheka/db/Trace.kt
package ru.android.zheka.db

import com.activeandroid.Model
import com.activeandroid.annotation.Column
import com.google.android.gms.maps.model.LatLng

class Trace : Model() {
    @Column(name = "name")
    var name: String? = null

    @Column(name = "start")
    var start: LatLng? = null

    @Column(name = "end")
    var end: LatLng? = null

    //use , for lat-lng delimiter and ; for points delimiter
    @Column(name = "data")
    var data: DataTrace? = null

    //public PolylineOptions data;
    override fun toString(): String {
        val util = UtilePointSerializer()
        val util1 = UtileTracePointsSerializer()
        return StringBuilder().append("name:")
                .append(name)
                .append(" start:")
                .append(util.serialize(start!!) as String)
                .append(" end:")
                .append(util.serialize(end!!) as String)
                .append("data:")
                .append(util1.serialize(data!!) as String).toString()
    }
}